lib/suspenders/app_builder.rb

```diff
@@ -221,9 +221,15 @@ def create_database
       bundle_command 'exec rake db:create db:migrate'
     end
 
-    def replace_gemfile
+    def replace_gemfile(path)
       remove_file 'Gemfile'
-      template 'Gemfile.erb', 'Gemfile'
+      template 'Gemfile.erb', 'Gemfile' do |content|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.